kong-plugin-request-transformer-1.3.0-0.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong-plugin-request-transformer"
-version = "1.2.2-0"
+version = "1.3.0-0"
```

Comment:
Since this is a bugfix, and it its not adding a feature, the "proper" version to use here would be `"1.2.3-0"`

This has a nice effect: it makes https://github.com/Kong/kong/pull/5119 not needed (the `~> 1.2` will take the latest which is not backwards compatible)

Reply:
This was merged as 1.3.0, right? So it does need the Kong PR for the version bump

Reply:
@hishamhm I just reverted this, opened a new PR as 1.2.3, and closed the PR on Kong for the version bump.